pom.xml

```diff
@@ -6,7 +6,7 @@
     <name>Cassandra Reaper</name>
     <groupId>com.spotify</groupId>
     <artifactId>cassandra-reaper</artifactId>
-    <version>0.5.1-SNAPSHOT</version>
+    <version>1.0.0-SNAPSHOT</version>
```

Comment:
Just to keep track that bringing in fault tolerance would mark the 1.0 milestone. 
Can be dealt with later of course.